sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -361,10 +361,18 @@ case class JsonTuple(children: Seq[Expression])
   // the fields to query are the remaining children
   @transient private lazy val fieldExpressions: Seq[Expression] = children.tail
 
+  // toString on null will throw NullPointerException so that return a very unlikely column name
+  private val nullFieldName = "__NullFieldName"
```

Comment:
`A field name given with constant null will be replaced with this pseudo field name. `